template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Solution/Solution1.sln

```diff
@@ -1,7 +1,7 @@
-﻿
+
```

Comment:
Apparently there are some whitespace changes here.  I don't know if that will affect anything but I think they should be reverted to be safe.

Reply:
@dsplaisted 
Thanks!
Done, whitespace changes reverted.